rdkit/ML/Scoring/Scoring.py

```diff
@@ -33,30 +33,30 @@ def CalcROC(scores, col):
     else:
       numInactives += 1
     TPR[i] = numActives  # TP
-    TNR[i] = numInactives  # TN
+    FPR[i] = numInactives  # FP
 
   # normalize, check that there are actives and inactives
   if numActives > 0:
     TPR = [1.0 * i / numActives for i in TPR]
   if numInactives > 0:
-    TNR = [1.0 * i / numInactives for i in TNR]
+    FPR = [1.0 * i / numInactives for i in FPR]
 
-  return [TNR, TPR]
+  return [FPR, TPR]
```

Comment:
While you are already changing this file, convert this into a namedtuple so that we can refer to the result by name
```
ROCcurve = namedtuple('ROCcurve', 'FPR,TPR')
...
return ROCcurve(FPR, TPR)
...
roc = CalcROC(scores, col)
FPR = roc.FPR
TPR = roc.TPR
```